lms/models/email_unsubscribe.py

```diff
@@ -0,0 +1,16 @@
+import sqlalchemy as sa
+
+from lms.db import BASE
+from lms.models._mixins import CreatedUpdatedMixin
+
+
+class EmailUnsubscribe(CreatedUpdatedMixin, BASE):
+    __tablename__ = "email_unsubscribe"
+    __table_args__ = (sa.UniqueConstraint("email", "tag"),)
+
+    id = sa.Column(sa.Integer(), autoincrement=True, primary_key=True)
+
+    tag = sa.Column(sa.UnicodeText(), nullable=False)
```

Comment:
This could be an array of tags but that's always more akward to query. One row per "tag" should be fine.